### DISCUSSION TITLE
How to edit quarto doc with Decap/Netlify CMS?

### DISCUSSION THREAD

**Discussion by drfurtado** [quarto-dev/quarto-cli#6376] [Q&A]
### Description

I created a blog with Quarto and deployed it at Neflify. Now, need to set up a directory in the poeject's root dir to unable editing it online via the Decap CMS. I went over the configuration process in [this link](https://decapcms.org/docs/add-to-your-site/) but wasn't able to configure it for Quarto. 

Does anyone have a step-by-step guide on how to set it up in the backend (root directory) of my Quarto blog?

Thanks.

**Comment by mcanouil** [comment]
I am not sure it's compatible as the source files are not published alongside the rendered files.
